app/javascript/components/KeyWordSearch.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import Button from 'react-bootstrap/lib/Button';
+import InputGroup from 'react-bootstrap/lib/InputGroup';
+import Form from 'react-bootstrap/lib/Form';
+import { faSearch } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+
+class KeyWordSearch extends React.Component{
+  constructor(props){
+    super(props);
```

Comment:
you don't have to call bind if you instead declare your component functions with the syntax

handleStudyLabel = (studyUrl) => {
...
}